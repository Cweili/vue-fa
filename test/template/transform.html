<p>
  <fa :icon="faMagic" size="4x" :rotate="90" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" rotate="30" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" rotate="-30" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" flip="vertical" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" flip="horizontal" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" flip="both" style="background: mistyrose"/>
  <fa :icon="faMagic" size="4x" flip="both" rotate="30" style="background: mistyrose"/>
</p>
