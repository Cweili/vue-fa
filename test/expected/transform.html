<p><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(1,1) rotate(90)">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(1,1) rotate(30)">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(1,1) rotate(-30)">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(1,-1) ">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(-1,1) ">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(-1,-1) ">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewBox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(-1,-1) rotate(30)">
        <path d="M14.1 463.3c-18.7-18.7-18.7-49.1 0-67.9L395.4 14.1c18.7-18.7 49.1-18.7 67.9 0l34.6 34.6c18.7 18.7 18.7 49.1 0 67.9L116.5 497.9c-18.7 18.7-49.1 18.7-67.9 0L14.1 463.3zM347.6 187.6l105-105L429.4 59.3l-105 105 23.3 23.3z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg></p>